docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/home_landing/field--node--field-landing-contents.html.twig

```diff
@@ -0,0 +1,42 @@
+{% set page_style = getHomeLandingStyle() %}
+{% if page_style == "ncid_without_title" %} 
```

Comment:
This should be with or without a title. Not just ncids_without_title. Also NCIDS is spelled wrong here.

Reply:
Done